pandas/core/arrays/categorical.py

```diff
@@ -1083,8 +1093,7 @@ def remove_unused_categories(self, inplace=False):
         cat._dtype = new_dtype
         cat._codes = coerce_indexer_dtype(inv, new_dtype.categories)
 
-        if not inplace:
-            return cat
```

Comment:
this doesnt get changed until we actually enforce the deprecation a few versions from now

Reply:
Ok